src/iter.rs

```diff
@@ -56,6 +48,18 @@ impl<'a> Iterator for HexToBytesIter<'a> {
     }
 }
 
+impl<'a> DoubleEndedIterator for HexToBytesIter<'a> {
+    fn next_back(&mut self) -> Option<Result<u8, HexToBytesError>> {
+        let lo = self.iter.next_back()?;
+        let hi = self.iter.next_back().unwrap();
+        Some(chars_to_hex(hi, lo))
+    }
+}
```

Comment:
Reminds me we should also override `nth` and `nth_back`.

Reply:
I added an issue.